run/types/DeviceWrapper.ts

```diff
@@ -933,12 +1000,124 @@ export class DeviceWrapper implements SharedDeviceInterface {
     await this.setValueImmediate(text, element.ELEMENT);
   }
 
+  public async getAttribute(
+    attribute: string,
+    elementId: string
+  ): Promise<string> {
+    return this.toShared().getAttribute(attribute, elementId);
+  }
+
+  public async radioButtonSelected(timeOption: DMTimeOption) {
+    try {
+      const radioButton = await this.findElementByXPath(
+        `//*[./*[@name='${timeOption}']]/*[2]`
+      );
+
+      const attr = await this.getAttribute("value", radioButton.ELEMENT);
+      if (attr === "selected") {
+        console.log("Great success - default time is correct");
+      } else {
+        throw new Error("Dammit - default time was not correct");
+      }
+    } catch (e) {
+      console.log(`Couldn't find radioButton ${timeOption}`);
+    }
+  }
+
+  public async sendImage(platform: SupportedPlatformsType, message: string) {
+    if (platform === "ios") {
```

Comment:
you should have the platform from the current class already, there is no need to pass the platform as an extra argument.

Reply:
replaced it with `this.isIOS()`